tests/MongoDB.Driver.Tests/PinnedShardRouterTests.cs

```diff
@@ -172,11 +172,15 @@ private DisposableMongoClient CreateDisposableClient(EventCapturer eventCapturer
             var client = DriverTestConfiguration.CreateDisposableClient((MongoClientSettings settings) =>
                 {
                     settings.ClusterConfigurator = c => c.Subscribe(eventCapturer);
-                    settings.LocalThreshold = TimeSpan.FromMilliseconds(1000);
+                    settings.LocalThreshold = TimeSpan.FromMilliseconds(5000);
```

Comment:
Note, the spec's test plan suggests one second as the local threshold:
https://github.com/mongodb/specifications/blob/master/source/transactions/tests/README.rst#mongos-pinning-prose-tests

Reply:
This change is probably not needed.

Reply:
Reverted!